runtimes/runtimes/standalone.ts

```diff
@@ -181,6 +182,61 @@ export const standalone = (props: RuntimeProps) => {
                         })
                     }),
                 getFileSize: path => stat(path),
+                getServerDataFolder: serverName => {
+                    const initializeParams = lspRouter.clientInitializeParams
+                    const clientSpecifiedLocation = initializeParams?.initializationOptions?.aws.clientDataFolder
```

Comment:
Can the aws component be null/undefined here?  Even if TypeScript says it's required, it's just parsed JSON.  Does that parsing validate against spec to ensure it exists?